default.properties

```diff
@@ -43,3 +43,25 @@ headerHeight=90
 # If different from default value "/header/", adapt
 # security-proxy/targets-mapping.properties accordingly
 headerUrl=/header/
+
+# Administrator email
+# Default email address used to send and receive mails in atlas, console and
+# extractorapp. See the corresponding properties files to override this email
+# address for specific needs.
+administratorEmail=georchestra@georchestra.mydomain.org
+
+# PostgreSQL database name
+# Default common PostgreSQL database for all geOrchestra modules
+pgsqlDatabase=georchestra
+
+# User to connect to PostgreSQL server
+# Default common PostgreSQL user for all geOrchestra modules
+pgsqlUser=georchestra
+
+# Password to connect to PostgreSQL server
+# Default common password of PostgreSQL user for all geOrchestra modules
+pgsqlPassword=georchestra
+
+# JDBC URL to connect to PostgreSQL server
+# Default common JDBC PostgreSQL URL for all geOrchestra modules
+jdbcUrl=jdbc:postgresql://localhost:5432/${pgsqlDatabase}
```

Comment:
Why not having also `psqlHost` and `psqlPort` defined here ?
This looks better IMO, at the (very mall) cost of having `jdbc:postgresql://${psqlHost}:${psqlPort}/${psqlDatabase}` in xml files...

Reply:
Yes, sure, I will add them.

Reply:
Done with d949125.

Reply:
From a end-user perspective, that would make sense to also remove `jdbcUrl` from default.properties. Don't you think ?

Reply:
OK.

Reply:
Done with https://github.com/georchestra/datadir/pull/149/commits/dfabfd556e5306f6d75b4a9317d98cd6b6eea3f7 and c5362dcdb3847cc2f48458aa92de0fd36c0f33f4.

Reply:
Note that the same comment would apply for `$publicUrl`, but it would mean changing in 28 places in datadir, plus 24 places in georchestra...

Reply:
I would vote +1 for the same replacement...

Reply:
OK. So much fun.

Reply:
:-)

Reply:
Sorry. Better now than never ;-)